include/dlaf/matrix/distribution.h

```diff
@@ -388,22 +452,15 @@ class Distribution {
     DLAF_ASSERT_HEAVY(i_begin <= i_end, i_begin, i_end);
     DLAF_ASSERT_HEAVY(0 <= i_begin && i_end <= global_nr_tiles_.get<rc>(), i_begin, i_end,
                       global_nr_tiles_.get<rc>());
-    DLAF_ASSERT(tilesPerBlock<rc>() == 1, "Multi Tile distribution block is not supported yet");
 
-    // Note the second assert is already done by the following calls.
     SizeType i_loc_begin = nextLocalTileFromGlobalTile<rc>(i_begin);
-    SizeType i_loc_last = nextLocalTileFromGlobalTile<rc>(i_end) - 1;
-    if (i_loc_begin > i_loc_last)
-      return 0;
-    SizeType l = local_size_.get<rc>();
-    SizeType nb = tile_size_.get<rc>();
-    SizeType nbr = std::min(nb, l - i_loc_last * nb);  // size of last local tile along `rc`
-    return (i_loc_last - i_loc_begin) * nb + nbr;
+    SizeType i_loc_end = nextLocalTileFromGlobalTile<rc>(i_end);
+    return localElementDistanceFromLocalTile<rc>(i_loc_begin, i_loc_end);
   }
 
-  /// \overlap localElementDistanceFromGlobalTile
+  /// \overload localElementDistanceFromGlobalTile
```

Comment:
overload simply group the function under a single doc (the one without overload)

Please check the output.

Reply:
Indeed, doing
```
/// \overload ...
LocalElementSize localElementDistanceFromGlobalTile(...)
```
gives an entry in brief summary at the top but no detailed entry further down. The function name is not a link.

Doing 
```
/// some text
LocalElementSize localElementDistanceFromGlobalTile(...)
```
gives an entry in the summary at the top with `some text` as the description in the summary, but no entry further down. The function name is a link that goes to the brief summary.

Doing 
```
/// some text
///
/// some more text
LocalElementSize localElementDistanceFromGlobalTile(...)
```
gives an entry in the summary at the top with no description, but an entry further down with `some text` and `some more text` in the description. The function name is a link that goes to the detailed description further down.

I think we want option 2?

Reply:
Option 2 or 3 is ok. `\overload` should be used only to share the same documentation.

Reply:
See https://github.com/eth-cscs/DLA-Future/pull/901/commits/9479d377f028fa385e8f77c4f2015e7a0ebd2b52. I've gone with option 2.